spec/serializers/recipe_show_serializer_spec.rb

```diff
@@ -55,6 +55,19 @@
     end
   end
 
+
+  describe 'rates' do
```

Comment:
```suggestion
  describe '#avg_rate' do
```
Please add also spec for `#rate_count`.